include/meter.hrl

```diff
@@ -1,8 +1,33 @@
 %% macros for meters
 %% register a meter for an application with opentelemetry:register_application_meter(AppName)
 
+-define(KIND_COUNTER, ot_counter).
+-define(KIND_UPDOWN_COUNTER, ot_updown_counter).
+-define(KIND_VALUE_RECORDER, ot_value_recorder).
+-define(KIND_VALUE_OBSERVER, ot_value_observer).
+-define(KIND_SUM_OBSERVER, ot_sum_observer).
+-define(KIND_UPDOWN_SUM_OBSERVER, ot_updown_sum_observer).
+
 -define(current_meter, opentelemetry:get_meter(?MODULE)).
 
+-define(new_counter(Meter, Name, Opts),
```

Comment:
yea, ok. I was going back and forth on this myself. Figuring if a module was including `opentelemetry_api/include/meter.hrl` it was clear where the metric macros are from and you definitely shouldn't be including multiple. But I'll change this, `ot_` isn't too long, so may as well.

Also, getting rid of `KIND_*`, so that is resolved on its own, thought I already deleted those.